app/split_integer.py

```diff
@@ -5,3 +5,6 @@ def split_integer(value: int, number_of_parts: int) -> list:
         parts.append(value // parts_left)
         value -= next_number
     return parts
+
+
+print(split_integer(0, 1))
```

Comment:
Consider adding error handling for cases where `number_of_parts` is zero or negative. This will prevent division by zero errors or logical errors in the function.